README.md

```diff
@@ -12,11 +12,13 @@ ocelot.social is a nonprofit social, action and knowledge network that connects
 * **Knowledge**: Read articles about interesting topics and find related posts in the **More Info** tab or by **Filtering** based on **Categories** and **Tagging** or by using the **Fulltext Search**.
 * **Action**: Don't just read about how to make the world a better place, but come into **Action** by following provided suggestions on the **Action** tab provided by other people or **Organisations**.
 
- [![Ocelot-Social](webapp/static/img/custom/welcome.svg)](https://ocelot.social)
+<p align="center">
+  <img src="webapp/static/img/custom/welcome.svg" alt="Ocelot-Social" width="40%" height="40%">
+</p>
 
 ## Live demo
 
-Try out our deployed [development environment](https://develop.human-connection.org/).
+__Try out our deployed [development environment](https://develop.human-connection.org/).__
```

Comment:
Do wee really want to keep this line? May be we can comment it out and wait until we have our own staging server.

Reply:
i just crossed it out for now, think thats ok